finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Row.scala

```diff
@@ -20,7 +20,7 @@ trait Row {
   /** The values for this Row. */
   val values: IndexedSeq[Value]
 
-  /**/
+  /** The value is to consider unsigned flag of field or not */
```

Comment:
haha awesome commit message :)

Sorry, I didn't mean to suggest that a comment was necessary here, only that there was a stray "/**/"